src/components/CommandTabs.astro

```diff
@@ -7,9 +7,10 @@ interface Props {
 	yarn?: string;
 	pnpm?: string;
 	cargo?: string;
+  bun?: string;
```

Comment:
It's cool to see this article and the site preview. Looks nice :) But I think you will have an easier time getting the article merged if you don't also include this unrelated change in the same PR.

Reply:
you're right, and now it isn't necessary anymore as we already implemented bun support on CommandTabs